lmdeploy/turbomind/supported_models.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from transformers import AutoConfig
+
+from lmdeploy.utils import get_logger
+
+logger = get_logger('lmdeploy')
+
+_SUPPORTED_ARCHS = dict(
+    # baichuan-7b
+    BaiChuanForCausalLM=True,
+    # baichuan2-7b, baichuan-13b, baichuan2-13b
+    BaichuanForCausalLM=True,
```

Comment:
why two `BaiChuanForCausalLM`?

Reply:
they changed the name from `BaiChuanForCausalLM` to `BaichuanForCausalLM` after `baichuan-7b`.
https://huggingface.co/baichuan-inc/Baichuan-7B/blob/5d86e56a58fe4a5b3292cd9bb7468afef6f93eab/config.json#L3
https://huggingface.co/baichuan-inc/Baichuan-13B-Chat/blob/e3e1498bb6b7fb0bcb5a65679c09b862d7c29301/config.json#L5